### PR TITLE
Add contenteditable attribute definition

### PR DESCRIPTION
**Pull request by bytestream**
See https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/contenteditable

### CODE REVIEW

File: library/HTMLPurifier/AttrDef/HTML/ContentEditable.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?php
+
+class HTMLPurifier_AttrDef_HTML_ContentEditable extends HTMLPurifier_AttrDef
```
Comment (ezyang) [inline library/HTMLPurifier/AttrDef/HTML/ContentEditable.php:3]: Why not just use Enum here?

**Comment by ezyang** [comment]
This is OK. If we put this in the library proper, it needs to be gated the same way textarea is gated.

**Comment by bytestream** [comment]
I only actually want to allow `false` so maybe `contenteditable="false` can be permitted, and `contenteditable=""` or `contenteditable="true"` only permitted when `HTML.Trusted` is enabled?

**Comment by bytestream** [comment]
The attribute has significance in wysiwyg editors. When htmlpurifier removes it, the editors behaviour differently :p

**Review by ezyang — APPROVED** [review]
this is not very idiomatic but whatever

**Pull request merged**